### DISCUSSION TITLE
I Flash BSX 8M cart and stay empty in Satellaview menu

### DISCUSSION THREAD

**Discussion by Beano34** [sanni/cartreader#1034] [Need help]
Hello, I have an OSCR HW5 V3 with the latest V14.3 update with the correct modules activated "clock generator and Satellaview" so I would like to flash a BSX 8M cart and it does not work...
However, I am reaching the end of the verification stage! I even tried to put back the original 1mb ROM and 32kb after-sales service and I systematically found myself with empty memory on the Satellaview. 
I checked the recommendations carefully before arriving at this result. I put a more powerful power supply, I flashed the backup with a FF.bin file to clear the memory, I shortened the name and path of the roms as much as possible.
But nothing works, I suspect that it activates CLK0 and CLK1? If yes, how to proceed? Grounding?

![68747470733a2f2f646c2e64726f70626f7875736572636f6e74656e742e636f6d2f732f307978347a396b71696c7331706e312f4857355f636c6f636b67656e2e6a70673f646c3d31](https://github.com/user-attachments/assets/305541ba-8f42-41de-ba16-599574c05d04)

Thanks you for your help !

**Comment by Beano34** [comment]
I answer myself it works with these ROMs [https://archive.org/search?query=+Satellaview+8M](url)
but do you know where I can find a writable ROM of BS Zelda or F-Zero Ace league? or a patch?

**Comment by Ryo-Seadas** [comment]
I have the same issue. I tried a dump of F-Zero grand prix 2 and it worked. The I tried with Kaizou Choujin Shubibinman Zero (from the same NO-Intro archive) and it results as an empty memory when I put back the cart into the Snes. I wonder why some dumps work and other don't..

**Comment by Beano34** [comment]
I think the real problem is the DRM included in the games. I tried with this software. 
https://github.com/devinacker/BSFlashManager
But I couldn't find a patch for the games. the ideal is probably to flash the BIOS of the BS-X cartridge with DRM-Free BIOS.

**Reply by sakman55** [reply]
Is the 8M cart truly empty after being written or is it only that the BS-X menu shows as empty?

The Satellaview ROMs have a header that counts the number of starts ("Limited Starts").  If the gameplay exceeds the number of permitted starts then the game/content will not show up on the BS-X menu.  The solution is to patch the Limited Starts byte in the header.

Original 8M dumps do not patch the Limited Starts byte as this tampers with the original data.

**Reply by Beano34** [reply]
Yes indeed I was faced with this problem I corrected this problem with the software. 
The games are displayed well and launch but after the intros it remains frozen with the screen written in "loading" in Japanese. 
Despite several successful flashes and checked at 100% of an 8M Card the result is the same black screen. For me is DRM problem

**Reply by Ryo-Seadas** [reply]
What games are you trying to launch? After using the BSFlashManager software to patch the rom (to get unlimited plays) now Kaizou Choujin Shubibinman Zero works perfectly

**Reply by Beano34** [reply]
All other programs work but popular games like "BS Zelda no Densetsu" and "BS F-Zero 2 Ace League" don't work for me.

**Reply by Ryo-Seadas** [reply]
I'm not a F-Zero expert, but if you mean F-Zero Grand prix 2 with the Ace League I'm pretty sure I flashed and played it succesfully. Same with a week 4 dump of Bs Zelda

**Reply by Beano34** [reply]
Ok can you give me the source or provide me with the ROM that works ? it may be my BS-X memory card or my OSCR which is causing the problem. thank you

**Comment by Beano34** [comment]
I think it's my clock generator which is defective I tried to follow this tutorial [https://github.com/sanni/cartreader/wiki/Calibrating-the-ClockGen](url) no matter where I put the wire it always remains on this screen...
![WhatsApp Image 2024-12-15 à 11 57 14_a35749c4](https://github.com/user-attachments/assets/c8ba1118-7116-4de9-be31-9f99e5b78169)
Do you have any ideas about replacing the generator? However, I checked the solders carefully, thank you

**Comment by Beano34** [comment]
Problem solved ! I changed the clock generator twice! because I received a defective one first ! and now I flash everything the first time !!! thx everyone !
